src/_locales/en/messages.json

```
@@ -186,6 +186,14 @@
     "message": "&QR code from link",
     "description": "The context menu entry shown for generating QR codes from a selected link with an access key."
   },
+  "contextMenuItemConvertPageURL": {
+    "message": "Generate QR Code for this Page",
+    "description": "Context menu item title for generating a QR code for the current page URL."
+  },
+  "contextMenuItemConvertPageURLAccessKey": {
+    "message": "&Generate QR Code for this Page",
```

```suggestion
    "message": "Generate QR code for this page",
    "description": "Context menu item title for generating a QR code for the current page URL."
  },
  "contextMenuItemConvertPageURLAccessKey": {
    "message": "&Generate QR code for this page",
```

AFAIK this is the capitalisation style we use in the translations for now, everywheere. Feel free to argue against this especially if you are a native speaker, but then changing this would need to be tackled in a new issue/PR and be consistent everywhere.

---

Thank you for the feedback! I've committed the suggested change.